uwsgi.ini

```diff
@@ -1,5 +1,5 @@
 [uwsgi]
-http-socket = :$(PORT)
+http-socket = :8080
```

Comment:
What's the reason for hardcoding the port?

Reply:
The reason I hardcoded the port is because when the app runs in the container and it tries to get the port through this [call](https://github.com/mitodl/lmod_proxy/blob/master/lmod_proxy/cmd.py#L14) nothing is returned and the value of port is `5000`